tests/func/test_checkout.py

```diff
@@ -676,37 +677,31 @@ def test_stats_on_removed_file_from_tracked_dir(tmp_dir, dvc, scm):
 
 
 def test_stats_on_show_changes_does_not_show_summary(
-    tmp_dir, dvc, scm, caplog
+    tmp_dir, dvc, scm, capsys
```

Comment:
Fixing these tests is making me think if I should just use logger in `ui.write()`. 🦊